handlers/server.go

```diff
@@ -25,6 +35,30 @@ type Config struct {
 	CircleCIToken string `json:"circle_ci_token"`
 }
 
+//webhook server
+type WebHookServer struct {
+	Address    string
+	Port       int64
+	ConfigFile string
+}
+
+//webhook handler
+func NewWebHookServer() *WebHookServer {
+	s := WebHookServer{
+		Address:    "0.0.0.0",
+		Port:       3000,
+		ConfigFile: "/root/bot/src/ci-bot/config.json",
+	}
+	return &s
+}
+
+//function to add flags
+func AddFlags(fs *pflag.FlagSet, s *WebHookServer) {
+	fs.StringVar(&s.Address, "address", s.Address, "IP address to serve, 0.0.0.0 by default")
+	fs.Int64Var(&s.Port, "port", s.Port, "Port to listen on, 3000 by default")
+	fs.StringVar(&s.ConfigFile, "config-file", s.ConfigFile, "Config file.")
```

Comment:
I don't like the config file because there is no way to track the configurations. Please transform all the configurations into Flags.